about.html

```diff
@@ -0,0 +1,121 @@
+        <li><a class="nav-item" href="./index.html#join-btn">Join</a></li>
+        <li><a class="nav-item" href="#partner">Sponsor</a></li>
+        <li><a class="nav-item" href="#">News</a></li>
+        <li class="campaign"><a class="nav-item" href="#">GW Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="about-main">
+      <p class="welcome">"Hi! Sharing world"</p>
+      <h1>10th GLOBAL WAQF <br>CONFERENCE 2022</h1>
+      <hr>
+      <p class="about-info">
+        Recognised as the most notable expression of Islamic philanthropy, Waqf is the voluntary, permanent, and
+        irrevocable dedication of a portion of one's wealth, in cash or kind, to Allah. Standing out as one of the
+        greatest achievements of Islamic civilization, Waqf is a philanthropic foundation, with the mission of
+        maximizing the value of endowments for the greater benefit of society.
+        This year the conference will serve as a platform to establish clarity, explore the boundaries of new waqf
+        developments, initiatives, and shed light on new thoughts, technologies, and regulations.
+      </p>
+      <p class="contact">Please contact us per Email for any further questions about Global Waqf Conference 2022!</p>
+      <span>
+        <a href="mailto:dr.nisaralig@gmail.com">dr.nisaralig@gmail.com</a>
+        <i class="fa-solid fa-chevron-right"></i>
+      </span>
+    </section>
+
+    <section class="about-logo">
+      <h2>Global Waqf Conference 2022 Logo</h2>
+      <hr>
+
+      <p>The undermentioned Logo of Global Waqf Conference was designed on 29 Novemver by Dr.Nisar Ali for use at this
+        site only.It is pertinent to mention that this is not the orignal Logo of GWC 2022.</p>
+      <div class="logo-container">
+        <img src="./Images/globalwaqf-logo.png" alt="Global Waqf Conference Logo">
+      </div>
+    </section>
+
+    <section class="past-events">
+      <h2>See the past Global Waqf Conferences</h2>
+      <hr>
+      <p>Take a look at the last two GWC events which took place in...</p>
+      <ul class="images-container">
+        <li class="seventh-event">
+          <div>
+            <h3>2019</h3>
+            <p>7th Global Waqf Conference</p>
+          </div>
+        </li>
+        <li class="eighth-event">
+          <div>
+            <h3>2020</h3>
+            <p>8th Global Waqf Conference</p>
+          </div>
+        </li>
+      </ul>
+
+
+    </section>
+
+    <section id="partner">
+      <h2>Partner</h2>
+      <hr>
+      <ul class="partner-logo">
+        <li><img src="./Images/partners/Finterra-logo.png" alt=""></li>
+        <li><img src="./Images/partners/mmu-logo.png" alt=""></li>
+        <li><img src="./Images/partners/fsmmu-logo.png" alt=""></li>
+        <li><img src="./Images/partners/isdb-logo.png" alt=""></li>
+        <li><img src="./Images/partners/iub-logo.png" alt=""></li>
+      </ul>
```

Comment:
- [ ] Great job giving your images an `alt` text 👍 . Kindly give this set of images a descriptive `alt` text 🙏 . This will help make your site more accessible ☺️

Reply:
[Add alt text to partners logo](https://github.com/MNisarAli/1st-Capstone-Conference-page/pull/1/commits/1e57b46479e11820b71311b0a3c519eb62f1e2d6)

Reply:
[Add alt text to partners logo](https://github.com/MNisarAli/1st-Capstone-Conference-page/pull/1/commits/1e57b46479e11820b71311b0a3c519eb62f1e2d6)